libzeth/serialization/r1cs_serialization.tcc

```diff
@@ -153,8 +153,9 @@ std::ostream &r1cs_write_json(
 
     out_s << "{\n";
     out_s << "\"scalar_field_characteristic\":"
-          << "\"Not yet supported. Should be bigint in hexadecimal\""
-          << ",\n";
+          << "\"0x" +
```

Comment:
Could pass `true` to `bitint_to_hex` instead of adding the prefix manually.

Reply:
Thanks @dtebbs - I switched to using this flag